modules/codegen/src/main/scala/com/twilio/guardrail/generators/Java/JacksonGenerator.scala

```diff
@@ -429,17 +434,22 @@ object JacksonGenerator {
           terms.filterNot(term => discriminatorNames.contains(term.propertyName))
 
         terms.foreach({
-          case ParameterTerm(propertyName, parameterName, fieldType, _, _, _, _) =>
+          case ParameterTerm(propertyName, parameterName, fieldType, _, _, _, _, constraints) =>
             val field: FieldDeclaration = dtoClass.addField(fieldType, parameterName, PRIVATE, FINAL)
             field.addSingleMemberAnnotation("JsonProperty", new StringLiteralExpr(propertyName))
+            for (constraint <- constraints) yield (constraint) match {
+              case Minimum(value) => field.addSingleMemberAnnotation("Min", new IntegerLiteralExpr(value.toString))
+              case Maximum(value) => field.addSingleMemberAnnotation("Max", new IntegerLiteralExpr(value.toString))
+              case _ => Unit
```

Comment:
I don't _think_ this is needed?

Reply:
Ah, I think if I leave this out then if I add more `PropertyConstraint` classes the compiler will warn that it might not include every possible match?

Reply:
As long as the parent trait of Minimum & Maximum is `sealed`, and those are the only two implementing classes, nope, not needed.  (And if it _was_ needed, you'd just have a blank case body there.)